config/environments/test.rb

```diff
@@ -6,6 +6,12 @@
 # and recreated between test runs. Don't rely on the data there!
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = true
```

Comment:
Please move it to initializers: `config/initializers/bullet.rb` with next context:

```ruby
if defined?(Bullet)
  Rails.application.configure do
    config.after_initialize do
      Bullet.enable = true

      Bullet.bullet_logger = true
      Bullet.console = true
      Bullet.raise = Rails.env.test?
    end
  end
end
```

Reply:
Added this file with config